spec/system/admin/order_spec.rb

```diff
@@ -420,7 +420,7 @@ def new_order_with_distribution(distributor, order_cycle)
 
     describe "viewing the edit page" do
       let!(:shipping_method_for_distributor1) do
-        create(:shipping_method, name: "Normal", distributors: [distributor1])
+        create(:shipping_method_with, :flat_rate, name: "Normal", amount: 12, distributors: [distributor1])
       end
       let!(:order) do
         create(:order_with_taxes, distributor: distributor1, ship_address: create(:address),
```

Comment:
I think the issue is that, creating an `order_with_taxes` (which has `completed_order_with_totals` as parent) will create a shipment for it too. So, later on, when... (continuing below)